### ISSUE TITLE
Duodenitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6080]
**Preferred term label:**
Duodenitis

**Synonyms**
Duodenal inflammation  

**Definition (free text, please give PubMed ID)**
Inflammation of the lining of the upper small intestine (duodenum)
PMID: 361306 

**Parent term (use hpo.jax.org/app)**
Abnormality of the small intestine

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 614328

**Gene association**
ADAM17 (OMIM: 603639)

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4724230/

**Comment by johannaest** [comment]
Hello,

Yes, this is another article about duodenitis. Can we add the term?

Regards 
Johanna
________________________________________
Von: Peter Robinson [notifications@github.com]
Gesendet: Montag, 17. August 2020 23:39
An: obophenotype/human-phenotype-ontology
Cc: Est, Johanna Ligaya; Author
Betreff: Re: [obophenotype/human-phenotype-ontology] Duodenitis (#6080)

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4724230/

**Comment by pnrobinson** [comment]
Yes, we will add it!

**Comment by pnrobinson** [comment]
adding as HP:0033117
